optaplanner-benchmark/src/main/java/org/optaplanner/benchmark/impl/result/SolverBenchmarkResult.java

```diff
@@ -269,6 +269,9 @@ public String getTotalScoreWithUninitializedPrefix() {
     }
 
     public String getAverageScoreWithUninitializedPrefix() {
+        if (getSuccessCount() == 0) {
+            return null; // return null, not String "null" so we can use a default value
```

Comment:
We should never use the string "null" anywhere - there really is never a valid case for it - so I wouldn't even bother adding this comment.

Merging already.